packages/pnpm-sync-lib/src/interfaces.ts

```diff
@@ -16,6 +16,13 @@ export interface ITargetFolder {
   folderPath: string;
 }
 
+export interface IFileStat {
+  file: string;
+  ino: number;
```

Comment:
`ino` is a very poor name that is inherited from ancient Unix systems.  If we don't rename it to follow coding conventions, then we should document it:

```suggestion
  /**
   * The inode number. A unique identifier for the file within the file system.
   */
  ino: number;
```

BTW GPT seems to think that:

> However, it's crucial to note that while NTFS has concepts akin to inodes, they differ in implementation and purpose. NTFS's file record numbers are unique within an NTFS volume but may change during defragmentation or other file system-level operations, unlike traditional Unix inodes, which generally remain constant over time.